Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh

```diff
@@ -0,0 +1,20 @@
+sudo apt-get -y update
+sudo apt-get install -y openjdk-8-jdk
+sudo apt-get -y update --fix-missing
+sudo apt-get install -y openjdk-8-jdk
```

Comment:
Not all Linux distros have apt-get. In other artifacts we've added conditional support for apt-get or yum. Here's an example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh